lib/constants/spacing.dart

```diff
@@ -27,6 +27,10 @@ library;
 
 import 'package:gap/gap.dart';
 
+/// Space between widgets in a Row.
+
+const double widthSpace = 5;
```

Comment:
I am moving to using Gap(5).

You should be using one of the context dependent constants in this file, not the widthSpace literal which should be removed.